tasks/src/main/java/ru/sberbank/school/task07/Counter.java

```diff
@@ -1,20 +1,20 @@
-package ru.sberbank.school.task07;
-
-
-import java.io.FileNotFoundException;
-
-/**
- * Считатель различных слов в файле.
- * <p>
- * Задание: Подсчитайте количество различных слов в файле.
- */
-public interface Counter {
-
-    /**
-     * Подсчёт различных слов в файле.
-     *
-     * @param pathToFile путь до файла
-     * @return количество различных слов
-     */
-    int count(String pathToFile) throws FileNotFoundException;
-}
+package ru.sberbank.school.task07;
+
+
+import java.io.FileNotFoundException;
+
+/**
+ * Считатель различных слов в файле.
+ * <p>
```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a867617329>*